tlatools/org.lamport.tlatools/javacc/tla+.jj

```diff
@@ -730,17 +730,15 @@ if (opArgs.kind != N_OpArgs) { ToolIO.out.println("Bug: not N_OpArgs node"); };
 
   private final String msgStackToString(ParseException e) {
     StringBuffer msg;
-    String name = " *unknown* (error occurred before module header)";
-    if (mn != null) name = mn.toString();
```

Comment:
Why can this check be removed?

Reply:
Who knows, it's what matches the generated files.

Reply:
Then let's not remove it.  Somebody probably put it there for a reason.

Reply:
No, somebody removed it from the generated files for a reason which is how SANY has been functioning all this time. The generated files are the current source of truth. This just matches tla+.jj to how SANY currently functions.

Reply:
FWIW: Removed from `TLAPlusParser.java` by yours truly in a5c252b70e463eec833ad0cf0e7f5af1d37d92eb nine years ago.